WebFeedGatewayPlugin.php

```diff
@@ -126,7 +127,8 @@ public function fetch($args, $request): bool
                     'latestDate' => $latestDate,
                     'feedUrl' => $request->getRequestUrl(),
                     'userGroups' => $userGroups,
-                    'includeIdentifiers' => $includeIdentifiers
+                    'includeIdentifiers' => $includeIdentifiers,
+                    'language' => str_replace(['_', '@'], '-', Locale::getLocale()),
```

Comment:
I've seen this in other places, I think it should be a method somewhere `toBcp47()`?!